CHANGELOG.md

```diff
@@ -44,11 +44,11 @@ Main (unreleased)
   - `otelcol.processor.tail_sampling` samples traces based on a set of defined
     policies from `otelcol` components before forwarding them to other
     `otelcol` components. (@erikbaranowski)
-  - `prometheus.integration.apache` collects metrics from an apache web server
+  - `prometheus.exporter.apache` collects metrics from an apache web server
     (@captncraig)
-  - `prometheus.integration.consul` collects metrics from a consul installation
+  - `prometheus.exporter.consul` collects metrics from a consul installation
     (@captncraig)
-  - `prometheus.integration.github` collects metrics from GitHub (@jcreixell)
+  - `prometheus.exporter.github` collects metrics from GitHub (@jcreixell)
 
 ### Enhancements
```

Comment:
Renaming these components is a breaking change, can you note the renamed components in the breaking changes section on line 13?

(It's outside of the changed hunk so I can't comment there directly)

Reply:
Is it also a breaking change even if the changes were not part of a release?

Reply:
I guess it is a breaking change only for the already released integrations, I will document those